libobs/obs-output.c

```diff
@@ -1907,8 +1922,9 @@ static void reset_packet_data(obs_output_t *output)
 	output->highest_video_ts = 0;
 	output->video_offset = 0;
 
-	for (size_t i = 0; i < MAX_AUDIO_MIXES; i++)
+	for (size_t i = 0; i < MAX_OUTPUT_AUDIO_ENCODERS; i++) {
```

Comment:
This is adding unnecessary brackets, though I forget what the project consensus is on brackets and single statement for loops.

Reply:
Yeah, I saw both... and it _seemed_ like they added braces around fors more often than not?

Reply:
But happy to do whatever

Reply:
Fixed in 69da75d32b061ff6a177f4177cbecbc06a024948